landoui/templates/stack/partials/landing-preview.html

```diff
@@ -87,9 +87,9 @@ <h3 class="StackPage-landingPreview-sectionLabel">Commits:</h3>
   </div>
 
   {% if target_repo.commit_flags %}
-  <h3 class="StackPage-landingPreview-sectionLabel">Flags:</h3>
+  <h3 class="StackPage-landingPreview-sectionLabel">Advanced settings:</h3>
   <div class="StackPage-landingPreview-section">
-      Flags checked off below will be appended to the commit message.
+      Flags selected below will be appended to the title of all commit messages in the stack.
```

Comment:
Hmmm I thought the behaviour was to append only to the latest descendant in the stack (ie the tip/head/whatever). I think that's the only changeset that matters for DONTBUILD, at least.

Reply:
The current behaviour is that it will be appended to all commit messages. This may be changed in the future.